src/actions/make/_make.lua

```diff
@@ -232,7 +232,7 @@
 ---------------------------------------------------------------------------
 
 	function make.objdir(cfg)
-		_x('  OBJDIR = %s', project.getrelative(cfg.project, cfg.objdir))
+		_x('  OBJDIR = %s', premake.esc(project.getrelative(cfg.project, cfg.objdir)))
```

Comment:
This file seems to mix and match `premake.` and `p.`, I was basing my usage off something else, which turns out is in another file entirely, woops! I might create a PR tomorrow that replaces *all* (within reason) occurrences of `premake.` with `p.`, for consistency.